From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: silence warning for chdir return value

This crashes with --enable-compiler-warnings:

shutdown.c:362: error: ignoring return value of 'chdir', declared with attribute warn_unused_result

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 util/reboot.c   |    6 +++++-
 util/shutdown.c |    6 +++++-
 2 files changed, 10 insertions(+), 2 deletions(-)

Index: upstart-0.5.0/util/shutdown.c
===================================================================
--- upstart-0.5.0.orig/util/shutdown.c
+++ upstart-0.5.0/util/shutdown.c
@@ -359,7 +359,11 @@ main (int   argc,
 
 
 	/* Give us a sane environment */
-	chdir ("/");
+	if (! chdir ("/")) {
+		nih_fatal (_("Could not chdir to /"));
+		exit(1);
+	}
+
 	umask (022);
 
 	/* Shutdown now? */
Index: upstart-0.5.0/util/reboot.c
===================================================================
--- upstart-0.5.0.orig/util/reboot.c
+++ upstart-0.5.0/util/reboot.c
@@ -269,7 +269,11 @@ main (int   argc,
 	kill (1, SIGTSTP);
 
 	/* Sync the disks */
-	chdir ("/");
+	if (! chdir ("/")) {
+		nih_fatal (_("Could not chdir to /"));
+		exit (1);
+	};
+
 	if (! no_sync) {
 		nih_info (_("Syncing disks"));
 		sync ();
